airbyte-protocol/protocol-models/src/main/java/io/airbyte/protocol/models/transform_models/FieldTransform.java

```diff
@@ -64,4 +67,8 @@ public UpdateFieldSchemaTransform getUpdateFieldTransform() {
     return updateFieldTransform;
   }
 
+  public Boolean getIsBreaking() {
```

Comment:
This should be `isBreaking()` instead of `getIsBreaking()`.

Also could this be a `boolean` instead of a `Boolean`?